reactDashboard/src/components/Footer/Footer.css

```diff
@@ -66,3 +71,31 @@
 footer {
   padding-bottom: 30px;
 }
+
+.darkmode footer {
+  background-color: #121212;
+  padding-bottom: 30px;
+}
+
+.darkmode footer .sitemap .entry {
+  color: #f2f2f2;
+  font-weight: normal;
```

Comment:
font-weight unnecessary here

Reply:
✅

Reply:
Misunderstanding - the font-weight is unnecessary in dark mode because it was in the base (lightmode). It needs to stay in the base selector.

Reply:
now back in base selector